extern/boostd-data/yugabyte/cassmigrate/20230913144459_deals_add_is_direct.go

```diff
@@ -14,7 +15,12 @@ func ts20230913144459_dealsAddIsDirectDealColumn(ctx context.Context, session *g
 	err := session.Query(qry).WithContext(ctx).Exec()
 
 	if err != nil {
-		return fmt.Errorf("creating new column IsDirectDeal: %w", err)
+		errorStr := strings.Split(err.Error(), ";")
+		if strings.Compare("code=2200", errorStr[0]) == 0 {
```

Comment:
I'd relax this directly to `strings.Contains`
```suggestion
		if strings.Contains(err.Error(), "code=2200") {
```

Reply:
Done